homework_1/zad2/taxLibrary/taxLibrary.go

```diff
@@ -0,0 +1,55 @@
+package taxLibrary
+
+import (
+	"github.com/pkg/errors"
+	"math"
+)
+
+type TaxClass struct {
+	Start      int
+	End        int
+	Percentage float64
+}
+
+func checkClassesDefinitions(classes []TaxClass) error {
+
+	for n, class := range classes {
+		if n != len(classes) - 1 {
+			if class.End != classes[n+1].Start {
```

Comment:
I think it also makes sense to test whether the End is >= Start within the same class